resident/resident-service/src/main/resources/bootstrap.properties

```diff
@@ -8,7 +8,7 @@ management.endpoints.web.exposure.include=info,health,refresh
 resident.service=resident
 config.server.file.storage.uri=${spring.cloud.config.uri}/${resident.service}/${spring.profiles.active}/${spring.cloud.config.label}/
 server.port=8099
-server.servlet.path=/resident/v1
+server.servlet.context-path=/resident/v1
```

Comment:
Why do we need this change ?

Reply:
the servlet context path is set in auth adapter. if we dont set it, that goes empty so we are not able to go unauthenticated. Now we removed servlet path added context-path to match it.